src/app/expand-collapse-all/expand-collapse-all.component.html

```diff
@@ -1,3 +1,20 @@
+<button mat-icon-button [matMenuTriggerFor]="menu" aria-label="Menu for additional settings">
+    <mat-icon>more_vert</mat-icon>
+</button>
+<mat-menu #menu="matMenu">
+    <button mat-menu-item (click)="refresh()">
+        <mat-icon>refresh</mat-icon>
+        <span>Refresh</span>
+    </button>
+    <button mat-menu-item (click)="toggle()">
+        <mat-icon>code</mat-icon>
+        <span>
+            <span [innerHTML]="showFullContent ? 'Hide' : 'Show'"></span>
```

Comment:
Nit: I think "showDomElementSource" is a more descriptive compared to showFullContent